cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -126,23 +135,59 @@ class CallSubqueryTest extends DocumentingTest {
         )))) { resultTable() }
     }
 
-    section("Aggregation and side-effects", "subquery-aggregation") {
-      p("""Subqueries can be useful to do aggregations for each row and to isolate side-effects.
-          #This example query creates five `Clone` nodes for each existing person.
-          #The aggregation ensures that cardinality is not changed by the subquery.
-          #Without this, the result would be five times as many rows.""".stripMargin('#'))
+    section("Cardinality and aggregation", "subquery-aggregation") {
+      p(
+        """Returning subqueries change the cardinality of the query: The results of each subquery call are collected and together they give the result of the
+          |enclosing query.""".stripMargin)
+      p("""The following example lists each connected person that each person has:""")
       query("""MATCH (p:Person)
               #CALL {
-              #  UNWIND range(1, 5) AS i
-              #  CREATE (c:Clone)
-              #  RETURN count(c) AS numberOfClones
+              #  WITH p
+              #  MATCH (p)-[:FRIEND_OF]-(c:Person)
+              #  RETURN c.name AS friend
               #}
-              #RETURN p.name, numberOfClones""".stripMargin('#'),
+              #RETURN p.name, friend""".stripMargin('#'),
       ResultAssertions(r => r.toSet should equal(Set(
-          Map("p.name" -> "Alice", "numberOfClones" -> 5),
-          Map("p.name" -> "Bob", "numberOfClones" -> 5),
-          Map("p.name" -> "Charlie", "numberOfClones" -> 5),
-          Map("p.name" -> "Dora", "numberOfClones" -> 5)
+          Map("p.name" -> "Alice", "friend" -> "Bob"),
+          Map("p.name" -> "Bob", "friend" -> "Alice"),
+        )))) { resultTable() }
+      p(
+        """The number of results of the subquery changed the number of results of the enclosing query: Instead of 4 rows, one for each node), there are
+          |now 2 rows which were found for Alice and Bob respectively. The nodes for Charlie and Dora are not returned.""".stripMargin)
+      p(
+        """Besides changing the set of matched nodes, which we can also express using a simple query, we can also use the subquery to aggregate on each of
+          |the matches of the enclosing query in isolation. As we get one result for each call of the subquery, the cardinality does not differ from the
+          |enclosing query's cardinality:""".stripMargin)
```

Comment:
I'd like to suggest:
``We can also use subqueries to perform isolated aggregations. In this example we count the number of relationships each person has. As we get one result for each call of the subquery, the number of rows is the same, before and after the `CALL` clause:``

Reply:
I generally like your version better. Two comments:

- Should we write "As we get one _row_" as you have suggested elsewhere to talk about rows? And possibly we should also write "invocation" of the subquery? (see other comment)
- We use  the word "cardinality" in the title and elsewhere, so I thought I could also use it here to unify the wording.

Reply:
`s/call/invocation/` :+1: 
Just did a quick search of the docs. The words _cardinality_ does not appear anywhere in the cypher manual, but the word _rows_ does multiple times :shrug:

Reply:
Right. I just realized that it was my change here, which introduced "cardinality" in the headline :see_no_evil: 

I now changed the headline to just read "Aggregations" instead of "Cardinality and aggregations".